app/controllers/protocols_controller.rb

```diff
@@ -0,0 +1,61 @@
+class ProtocolsController < ApplicationController
+  before_action :set_protocol, only: %i[show edit update destroy]
+
+  def index
+    @protocols = Protocol.all
+  end
+
+  def show
+  end
+
+  def new
+    @protocol = Protocol.new
+  end
+
+  def edit
+  end
+
+  def create
+    @protocol = Protocol.new(protocol_params)
+
+    respond_to do |format|
+      if @protocol.save
+        format.html { redirect_to @protocol, notice: 'Protocol was successfully created.' }
+        format.json { render :show, status: :created, location: @protocol }
```

Comment:
I don't think we need JSON replies in the admin UI

Reply:
Yeah, we'll remove these.